frontend/src/components/eam-configurate-group/eam-configurate-group.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { FC, useMemo } from 'react';
+import { Button, Input, Table } from 'components/common/common';
+import {
+  useAppDispatch,
+  useAppForm,
+  useEffect,
+  useAppSelector,
+  useState,
+} from 'hooks/hooks';
+import { DEFAULT_GROUP_PAYLOAD } from './common/constants';
+import { ButtonStyle, ButtonType, InputType } from 'common/enums/enums';
+import { getNameOf } from 'helpers/helpers';
+import {
+  EAMGroupConfigurate as groupsAction,
+  eam as eamActions,
+} from 'store/actions';
+
+import styles from './eam-configurate-group.module.scss';
+import { EAMGroupConfigurateRequestDto } from 'common/types/types';
+import { getRows, getColumns } from './helpers/helpers';
+
+const EAMConfigurateGroup: FC = () => {
+  const { control, errors, handleSubmit } =
+    useAppForm<EAMGroupConfigurateRequestDto>({
+      defaultValues: DEFAULT_GROUP_PAYLOAD,
+    });
+
+  const [selected_workers] = useState(new Set<string>([]));
+
+  const { id: tenantId } = useAppSelector(({ app }) => ({
+    id: app.tenant?.id,
+  }));
+
+  const dispatch = useAppDispatch();
+
+  useEffect(() => {
+    dispatch(
+      eamActions.getWorkers({
```

Comment:
you can't call `eam` actions here
you should have the same action for your page